x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_map.tsx

```diff
@@ -18,9 +18,20 @@ import { LocationMissingWarning } from './location_missing';
 const MapPanel = styled.div`
   height: 240px;
   width: 520px;
+  @media (max-width: 574px) {
+    height: 250px;
+    width: 100%;
+    margin-right: 0;
+  }
   margin-right: 20px;
 `;
 
+const EuiFlexItemTags = styled(EuiFlexItem)`
+  @media (max-width: 816px) {
+    order: 1;
```

Comment:
```suggestion
    flex-direction: column-reverse;
```
is that the same?

Reply:
flex-direction works on flex parent element, order is used for flex element.